examples/deprecation/deprecation.bal

```diff
@@ -0,0 +1,59 @@
+// The constant `CITY` is marked as deprecated using the `@deprecated` annotation and the markdown documentation `# # Deprecated`.
+# Define a constant to represent the city.
+# # Deprecated
+# This constant is deprecated as this information is not needed.
+@deprecated
+const string CITY = "COLOMBO";
+const string COUNTRY = "Sri Lanka";
+
+// The type `Address` is marked as deprecated using the `@deprecation` annotation.
+@deprecated
+public type Address CITY|COUNTRY;
+
+// The object `Data` is marked as deprecated using `@deprecated` annotation and the markdown documentation `# # Deprecated`.
+# The `Data` is a user-defined object.
+#
+# + firstName - This is the description of the `Data`'s `firstName` field.
+# + lastName - This is the description of the `Data`'s `lastName` field.
```

Comment:
The documentation of the type should focus on what this type is and what it is used for. The field documentation in this case can simply be something like `The first name of the person`

Reply:
Done